cwt/draft-ietf-ace-cbor-web-token.xml

```diff
@@ -912,7 +912,7 @@ a70175636f61703a2f2f61732e6578616d706c652e636f6d02656572696b7703
 
 	<section anchor="exampleKeysSymmetric128" title='128-bit Symmetric Key as Hex Encoded String'>
 	  <figure><artwork><![CDATA[
-8e82e68e61654ecb5a369fe8be7572dd
+8e82e68e61654ecb5a369fe8be7572
```

Comment:
I thought so to but it seem to be that I unintentionally added the extra 'dd' last time.
I think I will just generate a new key and update the encrypt example.